contents/join.md

```diff
@@ -0,0 +1,140 @@
+## Join {#sec:join}
+
+At the start of this chapter, we showed multiple tables and gave questions related to multiple tables.
+However, we haven't talked about combining tables yet, which we will do in this section.
+In `DataFrames.jl`, combining multiple tables is done via _joins_.
+Joins are extremely powerful, but might take some time to wrap your head around.
+It is not necessary to know the joins below by heart, the [`DataFrames.jl` documentation](https://DataFrames.juliadata.org/stable/man/joins/) and this book list them too.
+But, it's good to know that they exists.
+If you ever find yourself looping over rows in a DataFrame and and comparing it with other data, then you probably need one of the joins below.
+
+In @sec:dataframes, we've introduced the grades for 2020:
+
+```jl
+s = "grades_2020()"
+sco(s; process=without_caption_label)
+```
+
+Now, we're going to combine that with the information from 2021:
+
+```jl
+s = "grades_2021()"
+sco(s; process=without_caption_label)
+```
+
+To do this, we are going to use joins.
+The `DataFrames.jl` lists no less than seven kinds of joins.
+We show them all here, because they are all useful.
+This first mentioned one is `innerjoin`.
+Suppose that we have two datasets `A` and `B` with respectively columns `A_1, A_2, ..., A_n` and `B_1, B_2, ..., B_m` **and** one of the columns has the same name, say `A_1` and `B_1` are both called `:id`.
```

Comment:
I would say: Suppose that we have two datasets `A` and `B` with  columns `A_1, A_2, ..., A_n` and `B_1, B_2, ..., B_m` respectively **and** .... 

the `respectively` at the end, up to you.